src/controllers/auth.js

```diff
@@ -109,9 +109,24 @@ async function confirmed (req, res, next, client = FxAOAuthClient) {
   res.redirect(returnURL.pathname + returnURL.search)
 }
 
-function logout (req, res) {
-  req.session.destroy()
-  res.redirect('/')
+/**
+ * Controller to trigger a logout for user
+ * @param {object} req contains session.user
+ * @param {object} res redirects to homepage
+ */
+async function logout (req, res) {
+  log.debug('logout', req.session)
+  const subscriber = req.session?.user
+  log.debug('logout', subscriber?.primary_email)
```

Comment:
Do we want to remove or consolidate the debugs?  Are they silent in prod?

Reply:
Yes the debugs are silent in prod. I will actually change the email one to info